teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super('unknown', nil, parent_permission: true)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Kindly build your constructor well. You should require the necessary methods and they should be well parsed into the `super` method.

  ```
  def initialize(specialization, age, name = 'unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @specialization = specialization
  end
  ```